src/index.rst

```diff
@@ -616,6 +647,59 @@ Now ready to submit the workflow: ::
 
    $> bps submit test_sdf.yaml
 
+Submit a development workflow to USDF
+-------------------------------------
+
```

Comment:
Do we need to define what a "development workflow" is?  For example, that this section describes how to submit a workflow at USDF using own local software in addition to an LSST stack?

Reply:
I updated it to something like this:
Submit a workflow to USDF (Developer)
-------------------------------------

For developer to submit a workflow to S3DF with local software in addition to an LSST stack, please at first check `Submit a workflow to USDF`_.